testing/mongo-c-driver/APKBUILD

```diff
@@ -0,0 +1,56 @@
+# Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
+pkgname=mongo-c-driver
+pkgver=1.14.0
+pkgrel=0
+pkgdesc="Client library written in C for MongoDB"
+url="https://github.com/mongodb/mongo-c-driver"
+arch="all"
+license="Apache-2.0"
+makedepends="openssl-dev cyrus-sasl-dev icu-dev py3-sphinx cmake"
```

Comment:
```suggestion
depends_dev="${pkgname} cyrus-sasl-dev icu-dev cyrus-sasl-dev icu-dev" 
makedepends="${depends_dev} py3-sphinx cmake"
```

Reply:
Fixed.
I will mark this conversation as resolved.